README.md

```diff
@@ -90,6 +90,18 @@ D.get('year', date); // 2015
 
 ```
 
+###unixTime
+`String -> Date -> Number | Error`
+Returns the unix time of the supplied Javascript date object in the specified
```

Comment:
Yes, I think your right. We should probably rename the function `fromEpoch` then and the function `unixTime` can just be sugar for `fromEpoch('seconds')`.